### PR TITLE
New and Improved Marley Cake Adventure

### PR DESCRIPTION
**Pull request by kalez43**
This version uses a for loop and lists to move he user through the adventure.

### CODE REVIEW

**Comment by kalez43** [comment]
The function "response()" keeps iterating twice when I run the code, so the user has to input their answer (1 or 2) twice before moving on. Not sure why it is doing this

File: marley_cake_adventure.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        x += 1
+        user_input = input("Click here then press enter ==> ")
+        if user_input == "":
+            continue
+        else:
+            break
+    space()
+
+
+def response():
```
Comment (kalez43) [inline marley_cake_adventure.py]: This is where it's defined

File: marley_cake_adventure.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+move along the story and follow the prompts to choose the best answer for each dilemma. Make all the right choices and 
+eat some delicious cake!""")
+n = 0
+for part in story_lines:
+    stars()
+    print(part)
+    enter()
+    print(decisions[n])
+    space()
+    response()
```
Comment (kalez43) [inline marley_cake_adventure.py]: And this is where it is used in the for loop

**Comment by Bazarovay** [comment]
Merging

**Comment by kalez43** [comment]
Any thoughts on the response function?

**Pull request merged**